test/cql-pytest/test_describe.py

```diff
@@ -835,9 +835,12 @@ def new_random_keyspace(cql):
     options["class"] = random.choice(strategies)
     options["replication_factor"] = random.randrange(1, 6)
     options_str = ", ".join([f"'{k}': '{v}'" for (k, v) in options.items()])
+    extra = ""
+    if options["class"] == "NetworkTopologyStrategy" and options["replication_factor"] != 1:
+        extra = " and tablets = { 'enabled': false }"
```

Comment:
>  So why do we want to leave a test that attempts this? Shouldn't this test ALWAYS use RF=1?

The thing is that this place is not a test, but a helper function that creates "random" CREATE KEYSPACE statement for many other tests. For SimpleStrategy those tests may test something useful for RF=6 (*), for NetworkTopologyStrategy they should live with RF=1 only

(*) it's also a good question why those tests need RF={2, 3, 4, 5, 6}, maybe they can live with RF=3 only

Reply:
This is why I don't like kitchen-sink "help functions" in tests, they tend to have a life of their own and a bazillion options and nobody remembers why all of them are needed.
Specifically, which tests use this helper, and does any of them need  - for SimpleStrategy or NetworkTopologyStrategy - use use RF=6?

We just decided in this PR that that having one node and RF=6 is a bad idea and no real user will want to do this deliberately. We're forbidding this combination entirely with NTS and tablets because of implementation details, but we are also making a claim that beyond this implementation detail - this combination is simply not useful and no real user will want it. So why would we want to **test** this configuration?

Reply:
> Pretty much any

Doesn't it mean that we won't test tablets in most tests? That's not what we want, so I agree with @nyh that we should change the tests to use RF=1.

Reply:
Oh, or is this just or the tests which test "describe" ?

Reply:
This means that we will test tablets in all tests, but only when those tests are "lucky enough" to generate NetworkTopologyStrategy with replication factor of 1. With current input it's going to be ~1/12th of all runs.

Reply:
> Oh, or is this just or the tests which test "describe" ?

Yes, it's in test_describe.py and it only tests how DESC statement works for keyspace created with various parameters and options